packages/authentication/README.md

```diff
@@ -63,9 +63,9 @@ export class MyAuthStrategyProvider implements Provider<Strategy | undefined> {
     private metadata: AuthenticationMetadata,
   ) {}
 
-  value() : ValueOrPromise<Strategy> {
+  value() : ValueOrPromise<Strategy | undefined> {
     if (!this.metadata) {
```

Comment:
Please leave a comment above this line explaining why we're doing this when no metadata is defined (they didn't decorate the function with an auth method, so we shouldn't attempt to authenticate!)